src/js/view/view.js

```diff
@@ -928,6 +934,12 @@ define([
             _controlsTimeout = setTimeout(_userInactive, _timeoutDuration);
         }
 
+        function _toggleControls() {
+            _controlsHidden = !_controlsHidden;
+            utils.toggleClass(_playerElement, 'jw-flag-controls-hidden', _controlsHidden);
```

Comment:
Can we rename this jw-flag-controls-hidden?

.jwplayer.jw-flag-controls-hidden